parsec/core/gui/files_widget.py

```diff
@@ -888,7 +888,10 @@ async def _import_one(
         async with await trio.open_file(source, "rb") as f:
 
             # Getting the file name without extension (anything after the first dot is considered to be the extension)
-            name_we, suffixes = dest.name.str.split(".", 1)
+            name_splitted = dest.name.str.split(".", 1)
+            if len(name_splitted) == 1:
+                name_splitted.append("")
```

Comment:
This seems wrong: `foo` would get renamed as `foo (2).` instead of `foo (2)`

It would be better to do something like:
```python
suffixes: List[str]
name_we, *suffixes = dest.name.str.split(".")

[...]

# Create the new file name by adding the count ("myfile.txt" becomes "myfile (2).txt")
new_file_name = EntryName("."join([f"{name_we} ({count})", *suffixes)])
```

Reply:
Fixed! Good catch.